src/main.js

```diff
@@ -236,50 +207,80 @@ function displayHomePage() {
   removeClass(userActivityInputView, 'hidden');
 };
 
-function clearInput() {
+const clearInput = () => {
   selectedCategory = '';
   userSecondsInput.value = '';
   userAccomplishInput.value = '';
   userMinutesInput.value = '';
 };
 
-function clearButton() {
+const clearButton = () => {
   unhighlightCategory(meditateButton, meditateImage, 'meditate-click', 'meditate');
   unhighlightCategory(exerciseButton, exerciseImage, 'exercise-click', 'exercise');
   unhighlightCategory(studyButton, studyImage, 'study-click', 'study');
 };
 
-function clearCircle() {
+const clearCircle = () => {
   removeClass(circle, 'green');
   removeClass(circle, 'purple');
   removeClass(circle, 'red');
 };
 
-function clearErrorMessage() {
+const clearErrorMessage = () => {
   addClass(accomplishError, 'visibility');
   addClass(minutesError, 'visibility');
   addClass(secondsError, 'visibility');
 };
 
-function displayCreateNewButton() {
+const displayCreateNewButton = () => {
   activityTitle.innerText = 'Completed Activity';
   addClass(timerView, 'hidden');
   removeClass(createNewActivityView, 'hidden');
 };
 
-function completeActivity() {
+const completeActivity = () => {
   currentActivity.markComplete()
 };
 
-function disableButton(element) {
+const disableButton = (element) => {
   element.disabled = true;
   addClass(element, 'disabled');
 };
 
-function removeClass(element, rule) {
+const removeClass = (element, rule) => {
   element.classList.remove(rule);
 };
 
-function addClass(element, rule) {
+const addClass = (element, rule) => {
   element.classList.add(rule);
 };
+
+//event listeners
+startTimerButton.addEventListener('click', beginTimer);
```

Comment:
Way to move these event listeners to the bottom of the page, and changing the anonymous functions to arrow functions.